crates/rspack_plugin_javascript/src/visitors/dependency/common_js_import_dependency_scanner.rs

```diff
@@ -232,6 +242,60 @@ impl<'a> CommonJsImportDependencyScanner<'a> {
   pub fn walk_expression(&mut self, expr: &Expr) {
     expr.visit_children_with(self);
   }
+
+  pub fn check_ident(&mut self, ident: &Ident) {
+    if is_reserved_word_in_strict(ident.sym.as_str()) {
+      if self.is_strict {
+        self.errors.push(Box::new(create_traceable_error(
+          "SyntaxError".into(),
```

Comment:
I think you may use 'Javascript parsing error' here to align with other error messages.